specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -1652,6 +1652,90 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SecurityInsights/enrichment/ip/geodata/": {
+      "get": {
+        "x-ms-examples": {
+          "Get geodata for a single IP address": {
+            "$ref": "./examples/enrichment/GetGeodataByIp.json"
+          }
+        },
+        "tags": [
+          "Enrichment"
+        ],
+        "description": "Get geodata for a single IP address",
+        "operationId": "IPGeodata_Get",
```

Comment:
Enrichment_GetIPGeoData.  Does Enrichment give enough clarify and context if surfaced in SDK as it will be the operation class name.

Reply:
Yes, and this has already been approved by the ARM and the swagger/OpenAPI teams